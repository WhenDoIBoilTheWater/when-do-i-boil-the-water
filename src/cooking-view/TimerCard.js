import React from "react";

export class TimerCard extends React.Component {
	constructor(props) {
        super(props)
        this.state = {
        	recipe: this.props.recipe,
        	description: this.props.description,
        	when: this.props.when
        }
    }

    render() {
    	let date = Date.now();

    	console.log(date)

    	return(
    		<section>
	    		<p>{this.state.recipe}:</p>
	    		<h3>{this.state.description}</h3>
    		</section>
    	)
    }

}

export default TimerCard