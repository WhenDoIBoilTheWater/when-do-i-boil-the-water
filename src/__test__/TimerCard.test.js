import React from 'react';
import TimerCard from '../cooking-view/TimerCard.js';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <TimerCard />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
