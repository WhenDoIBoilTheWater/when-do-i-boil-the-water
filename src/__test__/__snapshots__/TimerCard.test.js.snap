// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="timer-card"
>
  <p />
  <p>
    Invalid Date
  </p>
  <br />
</section>
`;
